docker/README.md

```diff
@@ -0,0 +1,81 @@
+# Garage Docker
+
+Currently there's three versions of garage containers:
+  - ci: used for the continuous integration of garage in TravisCI and not
+    intended for final users.
+  - release: it's a published image based on the latest official release of
+    garage.
+  - local: it's an image built on top of the release with a copy of the files
+    in your local repository.
+
+## Release image
+
+The release image is published to be downloaded without the need of having
+a local repository. The command to run in the corresponding container can be
+specified through the argument RUN_CMD. To run an example launcher in the
+container, execute:
+```
+docker run --rm rlworkgroup/garage:release RUN_CMD="python examples/tf/trpo_cartpole.py"
```

Comment:
why do i need RUN_CMD? I can just put the command right on the terminal and it will be `$@` in the entrypoint

Reply:
Fixed!